pkg/custom-plugins/output_state.go

```diff
@@ -0,0 +1,33 @@
+package customplugins
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	apiv1 "github.com/leptonai/gpud/api/v1"
+)
+
+// StateOutputPrefix defines the prefix for the line,
+// where each plugin is expected to emit its state data in JSON format.
+// The JSON data schema for the state is defined in api/v1/types.go.
+const StateOutputPrefix = "GPUD_CUSTOM_PLUGIN_HEALTH_STATE:"
```

Comment:
we should make it shorter

Reply:
will update, and also add more e2e tests